### ISSUE TITLE
[Feature Request] Pity Timer on Open Pack screen.

### ISSUE THREAD

**Issue by Velken** [HearthSim/Hearthstone-Deck-Tracker#3015]
I have no idea how it would be coded, but I think it would be useful to have an overlay to tell you how many packs have been open since the last legendary from that.
Of course there would be the issue of opening a pack in other devices, then an option to manually add packs would also be good to add.

**Comment by judge2020** [comment]
This would best be a plugin, not everyone may want this and pack features are best not to be directly added to HDT.

**Comment by Velken** [comment]
Maybe a feature disabled by default? Like the Top deck % chance thingy.

**Comment by Velken** [comment]
so, there is a chance that this will become a default thing due to recent law changes in China: 
http://www.hearthpwn.com/news/2093-new-loot-box-regulations-in-china
tl;dr: Blizz will need to show all numbers of odds of getting cards from a packs, the  systems and algorithms and the pity timer and related stuff.

**Comment by speedytr** [comment]
This is a great idea as Blizzard said there have to be a legend every 40 pack so if HDT can track how many packs we open from each set we can keep on opening each specific set pack to reach legend.

**Comment by judge2020** [comment]
IIRC blizzard hasn't confirmed anything about a pity timer. We know there's a 40 pack legendary limit due to user reports.

i still feel a plugin would be more suited for this.

**Comment by jleclanche** [comment]
So you guys know, pack tracking on HSReplay.net can be turned on with some configuration modification. The packs are currently not displayed on the site but if this ever happens, you can start tracking them right now. Just let me know if you're interested and I'll help you turn it on.

**Comment by judge2020** [comment]
Likely going to be part of HSReplay, for those here looking for the china pack stats: https://www.reddit.com/r/hearthstone/comments/69dexs/china_announces_hearthstone_card_pack_rarity_odds/
